frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import {
+  // BaremetalHostRole,
+  // BaremetalHostStatus,
+  // getHostMachineName,
+  getHostBmcAddress,
+  getName,
+  getNamespace,
+  // getResource,
+  getUid,
+} from 'kubevirt-web-ui-components';
+
+// TODO(jtomasek): update import once models are moved to console-shared package
+// import { MachineModel, NodeModel } from '@console/internal/models';
+import { BaremetalHostModel } from '../models';
+
+// import { referenceForModel } from '@console/internal/module/k8s';
+
+import { ListHeader, ColHead, List, ListPage, ResourceRow } from '@console/internal/components/factory';
+
+import { ResourceLink } from '@console/internal/components/utils';
+// import { WithResources } from '@console/shared';
+
+import MachineCell from './machine-cell';
+
+const nameColumnClasses = 'col-lg-2 col-md-4 col-sm-6 col-xs-6';
+const statusColumnClasses = 'col-lg-2 col-md-4 hidden-sm hidden-xs';
+const machineColumnClasses = 'col-lg-3 visible-lg';
+const roleColumnClasses = 'col-lg-2 visible-lg';
+const addressColumnClasses = 'col-lg-2 visible-lg';
+
+const HostHeader: React.FC<React.ComponentProps<typeof ColHead>> = props => (
+  <ListHeader>
```

Comment:
Heads up that we're in the process of updating all tables to PF4 (#1465)

@priley86 FYI